src/scripts/main.js

```diff
@@ -1 +1,90 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  const timerId = () => {
+    reject(new Error('First promise was rejected'));
+  };
+
+  document.addEventListener('click', () => {
+    resolve();
+    clearTimeout(timerId);
+  });
+
+  setTimeout(timerId, 3000);
+});
+
+const secondPromise = new Promise(resolve => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+
+  document.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve();
+  });
+});
+
+const thirdPromise = new Promise(resolve => {
+  let isClickedLeft = false;
+  let isClickedRight = false;
+
+  document.addEventListener('click', () => {
+    isClickedLeft = true;
+
+    // Проверяем условие при каждом клике
+    if (isClickedLeft && isClickedRight) {
+      resolve();
+    }
+  });
+
+  document.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    isClickedRight = true;
+
+    if (isClickedLeft && isClickedRight) {
+      resolve();
+    }
+  });
+});
+
+firstPromise
+  .then(() => {
+    const message = document.createElement('div');
```

Comment:
create function with arguments, and pass success || warning, and message, and inside function create all logic that you do in then